uint8 check_id
---
uint8 job_nb
job[] jobs
